bundle/manifests/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -23,11 +23,11 @@ metadata:
     operatorframework.io/suggested-namespace: oco
     operators.operatorframework.io.bundle.channel.default.v1: alpha
     operators.operatorframework.io.bundle.channels: alpha
-    operators.operatorframework.io/builder: operator-sdk-v1.14.0
+    operators.operatorframework.io/builder: operator-sdk-v1.13.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/redhat-openshift-ecosystem/operator-certification-operator
     support: Red Hat
-  name: operator-certification-operator.v0.0.0-alpha
```

Comment:
Do you need to maybe rebase? This shouldn't be changing.

Reply:
This change automatically with make bundle probably. I didn't touch it. Not sure if I should change it manually.

Reply:
It seems that we should let this one as it is? Let me know if I need to change it.

Reply:
This is fine.